incubator/binding-openapi-asyncapi/src/main/moditect/module-info.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+module io.aklivity.zilla.runtime.binding.openapi.asyncapi
+{
+    requires com.fasterxml.jackson.core;
+    requires com.fasterxml.jackson.databind;
```

Comment:
Where is this dependency coming from, feels like we should be able to remove it.